system-tests/projects/e2e/cypress/e2e/subdomain.cy.js

```diff
@@ -16,16 +16,17 @@ describe('subdomains', () => {
     cy.visit('http://help.foobar.com:2292')
     cy.get('h1').should('contain', 'Help')
     cy.document().then((document) => {
-    // set cookies that are just on this subdomain
-    // and cookies on the superdomain
-    // and then regular cookies too
+      // set cookies that are just on this subdomain
+      // and cookies on the superdomain
+      // and then regular cookies too
       document.cookie = 'help=true; domain=help.foobar.com'
       document.cookie = 'asdf=asdf; domain=foobar.com'
       document.cookie = 'foo=bar'
     })
 
     cy.getCookies().then((cookies) => {
-      expect(cookies.length).to.eq(3)
+      // only cookies matching this subdomain should be yielded
+      expect(cookies.length).to.eq(2)
```

Comment:
and these are `help=true `& `foo=bar `?

Reply:
Yes, since they both match have domain `help.foobar.com`